fixtures/sprinkles/src/styles.css.ts

```diff
@@ -29,6 +30,26 @@ const responsiveStyles = createAtomicStyles({
       small: '10px',
       medium: '20px',
     },
+    background: {
+      red: {
+        vars: {
+          '--alpha': '1',
+        },
+        background: `rgb(255, 0, 0, var(--alpha, 1))`,
+      },
+    },
+    bgOpacity: {
+      none: {
+        vars: {
+          '--alpha': '1',
+        },
+      },
+      '10': {
+        vars: {
+          '--alpha': '0.1',
+        },
+      },
+    },
```

Comment:
This is an example of how I would like to be able to use this. The output here is:

```css
.background_red {
   --alpha: 1;
  background: rgba(255,0,0, var(--alpha, 1));
}

.bgOpacity_none {
  --alpha: 0;
}

.bgOpacity_10 {
   --alpha: 0.1;
}
```

This to me feels really powerful because the atoms no longer become a 1:1 relationship between the property and value. But instead become a combination of styles. This is just one immediate use case I have for this. But I can see more coming in the future.